src/features/user/TreeMapper/index.tsx

```diff
@@ -173,7 +171,7 @@ function TreeMapper(): ReactElement {
 
       <div id="pageContainer" className={styles.pageContainer}>
         {selectedLocation ? (
-          <PlantLocationPage {...TreeMapperProps} />
+          <InterventionPage {...TreeMapperProps} />
         ) : (
```

Comment:
Resolve the TS warning due to incompatible prop types passed in to InterventionPage